header.php

```diff
@@ -29,5 +29,13 @@
 <section id="main-container">
 
   <header id="header">
-    <h1><a href="<?php echo home_url(); ?>"><?php bloginfo('name'); ?></a></h1>
+    <h1 id="site-title"><a href="<?php echo home_url(); ?>">interglobal.vision</a></h1>
+    <div class="grid-row justify-end">
+      <div class="grid-item">
+        <a href="<?php echo qtranxf_convertURL('', 'en', false, true); ?>" hreflang="en">EN</a> / <a href="<?php echo qtranxf_convertURL('', 'es', false, true); ?>" hreflang="es">ES</a>
+        <?php if (in_array('ko', qtranxf_getSortedLanguages())) { ?>
```

Comment:
`qtranxf_getSortedLanguages()` returns an array with all values needed to setup the language toggle in the order that we configure. Could we use that array and iterate rather than only checking it for Korean please. In debug the hardcoding throws warnings. And array version is more flexible (for when we support Esperanto)

Reply:
oh i didnt realize there was a way to sort them. but duh the function name. ok i'll check that out